src/goto-symex/symex_main.cpp

```diff
@@ -319,6 +319,12 @@ void goto_symext::symex_step(reachability_treet &art)
         if (run_builtin(call, id.as_string()))
           return;
       }
+      else if (
+        has_prefix(id.as_string(), "c:@F@__VERIFIER_assert") && no_assertions)
+      {
+        cur_state->source.pc++;
+        return;
+      }
```

Comment:
Is this needed?

Reply:
I think the `-no-assertions` flag means the assertions don't need to be executed, does it?

Reply:
IIRC the no assertion flag is checked when creating the assert ssa, so this should not be needed